contrib/graph-gophers/graphql-go/option.go

```diff
@@ -65,9 +66,18 @@ func WithAnalyticsRate(rate float64) Option {
 	}
 }
 
-// WithOmitTrivial enables omission of graphql fields marked as trivial.
+// WithOmitTrivial enables omission of graphql fields marked as trivial. This
+// also opts trivial fields out of Threat Detection (and blocking).
 func WithOmitTrivial() Option {
 	return func(cfg *config) {
 		cfg.omitTrivial = true
 	}
 }
+
+// WithTraceVariables enables tracing of variables passed into GraphQL queries
+// and resolvers.
+func WithTraceVariables() Option {
+	return func(cfg *config) {
+		cfg.traceVariables = true
+	}
+}
```

Comment:
Looks like this new option is not covered by unit tests? Can we extend the tests that are in `graphql_test.go`?

Reply:
Atcually removed it, it only bloats this PR and isn't really called for.